### ISSUE TITLE
‘ptrdiff_t’ does not name a type

### ISSUE THREAD

**Issue by vigilantee-seek** [CAPTAIN-WHU/DOTA_devkit#6]
When I run the following command:
`python setup.py build_ext --inplace`
I meet with a gcc failed error. It is indicated in the log that 
`polyiou_wrap.cxx:2843:13: error: ‘ptrdiff_t’ does not name a type`
Can you help me with it?

**Comment by dingjiansw101** [comment]
Have you tried `swig -c++ -python polyiou.i`?
And what is your gcc version?

**Comment by vigilantee-seek** [comment]
Yeah, for sure I have tried `swig -c++ -python polyiou.i`. That's why I'm really confused.
My gcc version:
`gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.4)`

**Comment by mantou22** [comment]
您好，请问
807 331 800 324 817 309 823 316 ship 0
[(x1, y1), (x2, y2), (x3, y3), (x4, y4)] 分别代表的是什么地方的坐标位置？

**Comment by dingjiansw101** [comment]
四边形的四个顶点。具体参见论文，或者你可以可视化出来。

Dai wenxin <notifications@github.com> 于2018年10月11日周四 下午10:14写道：
